amplify/backend/hosting/S3AndCloudFront/template.json

```diff
@@ -39,7 +39,7 @@
                                     },
                                     "-",
                                     {
-                                        "Ref": "env"
+                                        "Ref": "AWS::AccountId"
```

Comment:
I wonder if it is related to the access denied error you encountered when running 'amplify publish'?